"make install" target broken on macOS in 0.3.1

```
==> make prefix=/usr/local/Cellar/no-more-secrets/0.3.1 install
install -d /usr/local/Cellar/no-more-secrets/0.3.1/share/man/man6
install -m644 -t /usr/local/Cellar/no-more-secrets/0.3.1/share/man/man6 nms.6 sneakers.6
install: illegal option -- t
usage: install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 file2
       install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 ... fileN directory
       install -d [-v] [-g group] [-m mode] [-o owner] directory ...
make: *** [install] Error 64
```

This was not broken in 0.3.0.

Build log here: https://bot.brew.sh/job/Homebrew%20Core/16712/version=sierra/consoleText

--- a/Makefile
+++ b/Makefile
@@ -48,7 +48,7 @@ clean:
 
 install:
 	install -d $(DESTDIR)$(mandir)/man6
-	install -m644 -t $(DESTDIR)$(mandir)/man6 nms.6 sneakers.6
+	install -m644 nms.6 sneakers.6 $(DESTDIR)$(mandir)/man6 
 	install -d $(DESTDIR)$(bindir)
 	cd $(BIN) && install * $(DESTDIR)$(bindir)
 

